addons/website_blog/static/src/js/website_blog.editor.js

```diff
@@ -72,6 +72,11 @@ if (!$('.website_blog').length) {
 
 const NEW_TAG_PREFIX = 'new-blog-tag-';
 
+// TODO Remove in master.
+for (const el of document.querySelectorAll(".o_wblog_social_links")) {
```

Comment:
This should surely be adapted in 16.0 to do the check in the iframe, right?
Thinking about that, the line below `$('.js_tweet, .js_comment').off('mouseup').trigger('mousedown');` seems wrong in 16.0 :thinking: it will search inside the webclient and not the iframe

Reply:
I can include that update during the forward port of this PR.

Reply:
Hopefully we don't forget :+1: